android/src/main/java/com/docrecog/scan/CameraActivity.java

```diff
@@ -421,6 +422,28 @@ protected void doOnResume() {
         keepScreenOnAwhile();
     }
 
+    public void saveLogFile() {
+            File file = new File(android.os.Environment.getExternalStorageDirectory().getPath()+"/Download", "AccuraEmirates.log");
+            String command = "logcat -f "+ file.getPath()+ " -v time *:V";
```

Comment:
why storing this to a file instead of reading the output of the command?
We don't want to add the EXTERNAL_STORAGE permission, users will not understand why suddenly we ask them for access to their device storage.

Reply:
it has been removed